### ISSUE TITLE
Koharu URL is blocked by cloudflare

### ISSUE THREAD

**Issue by CactusTheGuy** [keiyoushi/tachiyomi-extensions-source#5921]
### Source information

Koharu 1.4.8

### Source language

English

### Source new URL

https://niyaniya.moe/

### Other details

There is several redirect for koharu, for highest availability

https://seia.to/
https://shupogaki.moe/
https://hoshino.one/

### Acknowledgements

- [X] I have updated all installed extensions.
- [X] I have opened WebView and checked that the source URL has not changed yet.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.
